ntoskrnl/ke/bug.c

```diff
@@ -626,7 +626,7 @@ KiDisplayBlueScreen(IN ULONG MessageId,
         InbvResetDisplay();
 
         /* Display blue screen */
-        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_BLUE);
+        InbvSolidColorFill(0, 0, SCREEN_WIDTH - 1, SCREEN_HEIGHT - 1, BV_COLOR_GREEN);
```

Comment:
That's not doc.

Reply:
:(
I am going to start pulling from reactos/reactos master rather than minecraftchest1/reactos/master